README.md

```diff
@@ -1,2 +1,20 @@
-# Adapter
-this repo contains Adapter design pattern implementation using C#
+<div dir="rtl" align="right" >
+  
+  
+![Tuwaiq](https://i.ibb.co/SV2BSn5/tuwaiq.png)
+  
+  
+----
+# assignment
+
+Adapter design Pattern
+
+## UML Diagram
+
+<img src ="adapter_designPattern/UML.png" />
```

Comment:
Great UML and Idea Taghreed, but you should review the arrows in the UML, it needs some modification.
Check the arrows here
![UML](https://refactoring.guru/images/patterns/diagrams/adapter/structure-object-adapter-2x.png?id=03e8052e168c962d6bc3)